ttnn/cpp/ttnn/operations/moreh/moreh_dot/moreh_dot.hpp

```diff
@@ -5,17 +5,18 @@
 
 #pragma once
 #include "ttnn/decorators.hpp"
+#include "ttnn/operations/core/compute_kernel/compute_kernel_config.hpp"
 namespace ttnn::operations::moreh::moreh_dot {
 
 struct MorehDot {
-
     static Tensor invoke(
-        const ttnn::Tensor& input_tensor_a,
-        const ttnn::Tensor& input_tensor_b,
-        const std::optional<DataType> dtype,
-        const std::optional<MemoryConfig> &memory_config);
+        const Tensor& input_a,
```

Comment:
want to highlight the inconsistency. I see that usually in your ops you add support for optional preallocated output tensor. but this operation does not have that.

Reply:
@thanhnguyen-moreh please support optional output, I will review for last time.